pkg/api/norman/customization/aks/listers.go

```diff
@@ -248,6 +249,7 @@ func listClusters(ctx context.Context, cap *Capabilities) ([]byte, int, error) {
 					return nil, http.StatusInternalServerError, fmt.Errorf("could not parse virtual network ID")
 				}
 				tmpCluster.ResourceGroup = match[1]
+				tmpCluster.Location = to.String(cluster.Location)
```

Comment:
Could we put this below line 244 so the location gets set regardless of whether `cluster.ID != nil`?

Reply:
done